### PR TITLE
Use appropriately sized cover images in add covers modal

### LINKED ISSUE #5217: Add cover thumbnails have gone from small to miniscule.
Adding a cover to a work opens a preview of available covers. Since deployment on 27-05-2021, these thumbnails are too small to be of much use. Prior to this they were larger, but only the left half could be seen. Prior to the new books page the covers were fully displayed.
### Evidence / Screenshot (if possible)
Screenshot 2021-05-28 at 08 29 28

### Relevant url?
https://openlibrary.org/works/OL1841317W/Star_Wars_-_New_Jedi_Order_-_The_Final_Prophecy/add-cover

### Steps to Reproduce

1. Go to ...any work url
2. Do ...append with "/add-cover" to open cover editor

* Actual: see screenshot
* Expected: covers should be large enough to see detail and displayed in full

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** Version 90.0.4430.212 (Official Build) (x86_64)
- **Operating system?** MacOS Big Sur
- **Environment (prod/dev/local)?** prod

### PR DESCRIPTION
**Pull request by jamesachamp**
Closes #5217

Sets the maximum number of covers displayed at once in the add covers modal carousel to three.

### Technical

Explicitly passes number of books to be displayed to Carousel initialization function.

### Screenshot

add-cover-3-max

### Stakeholders

@seabelis

### CODE REVIEW

**Comment by jamesachamp** [comment]
@seabelis Are you ok with only having three covers displayed at a time?  I tried displaying four with this solution, but they were still a little small:

add-covers-4-max

The three cover solution is on testing if you want to try it out.

File: openlibrary/templates/covers/add.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
                         <div class="carousel-section">
                             <div id="covers" class="carousel-container carousel-container-decorated carousel--minimal">
                                 <div id="popcovers" class="carousel carousel--progressively-enhanced"
-                                    data-config="$json_encode(['#popcovers'])">
+                                    data-config="$json_encode(['#popcovers', 3, 3, 3, 3, 2, 1])">
```
Comment (jdlrobson) [inline openlibrary/templates/covers/add.html:37]: Yep that's exactly the right approach here.

```
     * @param {Number} [a] number of books to show (default)
     * @param {Number} [b] number of books to show @1200px or more
     * @param {Number} [c] number of books to show @1024px or more
     * @param {Number} [d] number of books to show @600px or more
     * @param {Number} [e] number of books to show @480px or more
     * @param {Number} [f] number of books to show @360px or more
   ```

**Comment by seabelis** [comment]
@jamesachamp Yes, three is fine.

**Comment by mekarpeles** [comment]
Given @seabelis + @jdlrobson's feedback, biasing towards merging

**Pull request merged**